src/status_im/ui/screens/add_new/new_public_chat/view.cljs

```diff
@@ -51,12 +51,12 @@
      [list/item-icon {:icon      :icons/forward
                       :icon-opts {:color :gray}}]]]])
 
-(def default-public-chats ["status" "openbounty" "ethereum"])
+(def default-public-chats ["status" "openbounty" "edcon" "ethereum"])
 
 (views/defview new-public-chat []
   (views/letsubs [topic [:get :public-group-topic]
                   error [:get :public-group-topic-error]]
-    [react/view styles/group-container
+    [react/keyboard-avoiding-view styles/group-container
```

Comment:
Does this fix some other issue?

Reply:
yes, annoying one

Reply:
but we need this new topic for conference , let's say we need this fix for conference too :)